packages/server-wallet/.vscode/launch.json

```
@@ -39,6 +39,7 @@
       "internalConsoleOptions": "neverOpen",
       "env": {
         "NODE_ENV": "test"
+        //"SHOW_VERBOSE_GANACHE_OUTPUT": "true"
```

@snario, good call, but need a comma on the line above
```suggestion
        "NODE_ENV": "test",
        "SHOW_VERBOSE_GANACHE_OUTPUT": "false"
```